### PR TITLE
✨ Integrate AdSensor network into AMP

### PR DESCRIPTION
**Pull request by jsalgueiro**
# Description:

- Integrate AdSensor network into AMP

### CODE REVIEW

**Comment by jsalgueiro** [comment]
I signed it

File: ads/adsensor.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/adsensor.js]: nit: Incorrect Copyright year 😄

File: ads/adsensor.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/adsensor.md]: nit: Incorrect Copyright year

Comment (jsalgueiro) [reply ads/adsensor.md]: Fixed! :)

File: ads/adsensor.js (hunk 1 above)
Comment (jsalgueiro) [reply ads/adsensor.js]: Fixed.

**Comment by jsalgueiro** [comment]
@torch2424  Its seems that I still have problems with the CLA but I already have signed the document. Do you know what's the issue?
Thanks in advance for your support.

**Comment by jsalgueiro** [comment]
Ok, the agreement was only signed yesterday so I think I have to wait a few days for the document to be accepted.
Meanwhile, I have made a rebase to change the commits author email and now I think all the commits have the same email with which agreement was signed.
Thanks.

**Comment by torch2424** [comment]
@jsalgueiro Nice, thanks for doing that rebase!

Even though the agreement was just signed, should take longer than an hour I think? Can you try doing a small whitespace change to get the bot to run again? (though it should run on comments).

Thank you! 😄

**Comment by mrjoro** [comment]
@jsalgueiro I did a manual check for your CLA status and it looks like you haven't yet signed it.

Did you sign the individual CLA or did you sign a corp CLA for AdSensor?  The corp ones can take a little bit longer to go through since they require some manual steps.

(Once it is signed you can get the CLA check to run again on the PR by adding a comment mentioning @googlebot.)

**Comment by jsalgueiro** [comment]
@torch2424  and @mrjoro  thank you for your support.
I've signed the corp CLA for AdSensor but it was refused because I don't have a title with the required authority in the company.
Today I will check internally who can have that responsibility and we will sign again the CLA.

**Comment by torch2424** [comment]
@jsalgueiro Awesome! Thank you! Please let me know how that goes 😄

**Comment by jsalgueiro** [comment]
@torch2424 The CLA was signed and approved! Now I'm covered by the corporate agreement and I think that everything is good to go.

**Comment by torch2424** [comment]
@jsalgueiro My apologies! Was a busy end of the week. Thanks for signing the CLA, doing a final review now. 😄

**Comment by jsalgueiro** [comment]
@torch2424 You're right!! It's my intention to implement renderStart but I've not implemented yet and I forgot to remove it from the configuration. 
I have pushed a commit removing it from the configuration.

Thank you.

**Comment by torch2424** [comment]
@jsalgueiro Ah okay that makes sense. 

If I could say, we highly recommend implementing `renderStart` if you can before this gets pulled in. It really helps us serve ads fast to our users. 👍 

However, I understand if it's not on your current timeline, and if not, I can merge this in.

Let me know, thanks! 😄

**Comment by jsalgueiro** [comment]
@torch2424 Ok, I've followed your advice and implemented the `renderStart`. Can you check if everything is ok now?

Thank you for your support 👍

**Comment by torch2424** [comment]
@jsalgueiro Merged this in. Just also wanted to say, thank you very much for your patience, thank you for working with us especially with the `renderStart`, and thank you very much for your contribution to AMP! 😄 

Have a nice day! Feel free to reach out with any questions.

**Pull request merged**